### PR TITLE
Add timing/logging to simple assembly CLI, rename it to assembly CLI.

### PR DESCRIPTION
**Pull request by hickst**

### CODE REVIEW

File: src/main/scala/edu/arizona/sista/reach/AssemblyCLI.scala [hunk 1]
```diff
@@ -2,173 +2,72 @@ package edu.arizona.sista.reach
...
+    for (file <- files) {
       val paperId = FilenameUtils.removeExtension(file.getName)
-      val startTime = AssemblyCLI.now // start measuring time here
       val startNS = System.nanoTime
+      val startTime = AssemblyCLI.now
+
+      FileUtils.writeStringToFile(logFile, s"${startTime}: Starting $paperId\n", true)
+      if (verbose)
+        println(s"  ${nsToS(startNS, System.nanoTime)}s: $paperId: starting reading")
+
```
Comment (myedibleenso) [inline src/main/scala/edu/arizona/sista/reach/AssemblyCLI.scala]: Sure would be great to have working logger to handle all of this stuff...

**Comment by myedibleenso** [comment]
Does this have any connection to #173?

**Comment by hickst** [comment]
yes, that's why GitHub says: 
hickst wants to merge 1 commit into master from issue173 
at the top of the pull request ;)

**Comment by myedibleenso** [comment]
Oops!  Didn't notice that.  Yeah, now I see that you named your branch after that issue...

**Pull request merged**